Slexy.js

```diff
@@ -0,0 +1,106 @@
+{
+	"translatorID": "3a4c6510-8082-4c99-9fa4-458a9cb29795",
+	"label": "Slexy",
+	"creator": "febrezo",
+	"target": "^https?://slexy\\.org/view/.+",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-09-15 22:19:16"
+}
+
+/*
+	Slexy Translator
+	Copyright (C) 2016 Félix Brezo, felixbrezo@gmail.com
+	
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the Affero GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+	
+	You should have received a copy of the Affero GNU General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	return "webpage";
+}
+
+function doWeb(doc, url) {
+	var resourceType = detectWeb(doc, url);
+
+	// Creating the item
+	var newItem = new Zotero.Item(resourceType);
+
+	//Setting common data:
+	newItem.websiteTitle = "Slexy";
+	newItem.websiteType = "Paste Site";
+	newItem.url = url;
+	
+	// Grabbing paste's metadata from "data" object
+	var info = ZU.xpathText(doc, '//table[@id="data"]//b').split(',');	
```

Comment:
That's unnecessarily fragile.
You can just go through the four table cells:
`//table[@id="data"]/tbody/tr[1]/td[1]/b`    (and 1/2, 2/1, 2/2 for tr and td respectively) to get at the four entries. That way, e.g. they don't break when one contains a comma.